addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -208,6 +208,9 @@ def full_filter_invl_to_apply(invoice_line):
             tax_details_info['NoSujeta']['ImportePorArticulos7_14_Otros'] = round(sign * base_amount_not_subject, 2)
         if not invoice.company_id.currency_id.is_zero(base_amount_not_subject_loc) and invoice.is_sale_document():
             tax_details_info['NoSujeta']['ImporteTAIReglasLocalizacion'] = round(sign * base_amount_not_subject_loc, 2)
+        if tax_details_info.get('NoSujeta'):
+            is_oss = self._has_oss_taxes(invoice)
+            tax_details_info['NoSujeta']['Causa'] = 'IE' if is_oss else 'RL'
```

Comment:
We can not put it in the SII json... (as that json might be sent to the gov)

Reply:
Indeed. The tests are failing.

I've updated the fix.

Thanks.